util/custom-response/src/main/java/com/brother/customresponse/enums/ActiveStatus.java

```diff
@@ -0,0 +1,21 @@
+package com.brother.customresponse.enums;
+
+public enum ActiveStatus {
+    ACTIVE(1),
+    ARCHIVE(2),
+    DELETE(3),
+    OPEN(4),
+    CLOSED(5),
+    PREVIOUS(6);
```

Comment:
not need to use semicolon at the last element of enum

Reply:
> not need to use semicolon at the last element of the enum

It is mandatory to use semicolons at the last constant(value).